### ISSUE TITLE
Screenshot not available

### ISSUE THREAD

**Issue by raffaem** [JakeStanger/ironbar#378]
The screenshot in the README is not available

**Comment by Sneethe** [comment]
In case anyones curious about what it looked like.
 https://web.archive.org/web/20231024201237if_/https://camo.githubusercontent.com/b610261ef7a0d32842bcb4bbc6ea6b252573a991d2b68396b62fefd3329cc1ba/68747470733a2f2f662e6a7374616e6765722e6465762f6769746875622f69726f6e6261722f6261722e706e673f726177

**Comment by yavko** [comment]
> In case anyones curious about what it looked like. [web.archive.org/web/20231024201237if_/https://camo.githubusercontent.com/b610261ef7a0d32842bcb4bbc6ea6b252573a991d2b68396b62fefd3329cc1ba/68747470733a2f2f662e6a7374616e6765722e6465762f6769746875622f69726f6e6261722f6261722e706e673f726177](https://web.archive.org/web/20231024201237if_/https://camo.githubusercontent.com/b610261ef7a0d32842bcb4bbc6ea6b252573a991d2b68396b62fefd3329cc1ba/68747470733a2f2f662e6a7374616e6765722e6465762f6769746875622f69726f6e6261722f6261722e706e673f726177)

This screenshot is pretty old, maybe a new screenshot with newer functionality would be cool

**Comment by JakeStanger** [comment]
Yeah, happy to provide some new ones once I'm up and running again. Of course, happy for anybody who has anything they'd like to show off to ]submit theirs](https://github.com/JakeStanger/ironbar/discussions/categories/show-and-tell) and I'll consider using that in the README/docs too :)

**Comment by JakeStanger** [comment]
WD have shipped my drives at long last! I'll have screenshots back up likely mid next week.

**Comment by JakeStanger** [comment]
900Mbps is a lot faster than I thought. Back up and I've updated the screenshot :)

**Comment by JakeStanger** [comment]
Nevermind I'm getting drive errors with the replacements :/ 

Will try and sort out something temporary in the meantime.
